tox.ini

```diff
@@ -4,7 +4,7 @@ envlist=py36,py37,pylint,flake8,safety,bandit,docs
 [travis]
 python =
   3.6: py36
-  3.7: py37,pylint,flake8,safety,bandit,docs
```

Comment:
Why delete safety? @avara1986

Reply:
pipenv had got a bug: https://github.com/pypa/pipenv/issues/4147

Now is closed, i try again if it's works :)